src/Swarm/TUI/Inventory/Sorting.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |
+-- Module      :  Swarm.TUI.Inventory.Sorting
+-- Copyright   :  Brent Yorgey
```

Comment:
It's actually a little silly to say that I hold the copyright to this file that I didn't write.  By contributing to this repository you're agreeing to license your contributions under a BSD open-source license, of course, but you're certainly not actually giving me the copyright.  You should either put your own name, or maybe we should discuss something more systematic like putting "Swarm contributors", or not listing a copyright field at all.

Reply:
I'd prefer omitting it entirely.  Maximize information density in the source code by eliminating redundancies.  The root-level copyright doc certainly covers all of the source files.